.github/workflows/test.yml

```diff
@@ -10,7 +10,20 @@ concurrency:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - id: "linux-aarch64"
+            os: "ubuntu-latest"
+          - id: "linux-amd64"
+            os: "ubuntu-20.04"
+          - id: "darwin-aarch64"
+            os: "macos-latest"
+          - id: "darwin-amd64"
+            os: "macos-13"
+          - id: "windows-amd64"
+            os: "windows-2022"
```

Comment:
Any reason to limit the windows version here?

Reply:
Primarily just preparation for the ARM version of windows